integration_tests/loop_04.py

```diff
@@ -0,0 +1,6 @@
+from ltypes import i32
+j: i32
+i: i32
+for i in range(5):
+    j = i
+    print("j: ", j)
```

Comment:
Register it in https://github.com/lcompilers/lpython/blob/main/integration_tests/CMakeLists.txt with `cpython`, `llvm`.

Reply:
@anutosh491 was there any issue?